account/templates/account/dashboard.html

```diff
@@ -7,25 +7,263 @@
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              ...
+            </div>
+            <div class="modal-footer">
+              <button type="button" class="btn btn-danger" data-dismiss="modal">Cut</button>
+              
+            </div>
+          </div>
+        </div>
+      </div>
+
+
+
+      <div class="modal fade" id="receivingmodal" tabindex="-1" role="dialog" aria-labelledby="exampleModalLabel" aria-hidden="true">
+        <div class="modal-dialog" role="document">
+          <div class="modal-content">
+            <div class="modal-header">
+              <h5 class="modal-title" id="receivingdetails">Modal title</h5>
+              <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                <span aria-hidden="true">&times;</span>
+              </button>
+            </div>
+            <div class="modal-body">
+              ...
+            </div>
+            <div class="modal-footer">
+              <button type="button" id="acceptcall" class="btn btn-primary" data-dismiss="modal">Accept</button>
+              <button type="button" class="btn btn-danger" data-dismiss="modal">Reject</button>
+              
+            </div>
+          </div>
+        </div>
+      </div>
+
+
 </body>
-<script src="https://ajax.googleapis.com/ajax/libs/jquery/3.6.0/jquery.min.js"></script>
+<script src="https://code.jquery.com/jquery-3.2.1.slim.min.js" integrity="sha384-KJ3o2DKtIkvYIK3UENzmM7KCkRr/rE9/Qpg6aAZGJwFDMVNA/GpGFF93hXpG5KkN" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/popper.js@1.12.9/dist/umd/popper.min.js" integrity="sha384-ApNbgh9B+Y1QKtv3Rn7W3mgPxhU9K/ScQsAP7hUibX39j7fakFPskvXusvfa0b4Q" crossorigin="anonymous"></script>
+    <script src="https://cdn.jsdelivr.net/npm/bootstrap@4.0.0/dist/js/bootstrap.min.js" integrity="sha384-JZR6Spejh4U02d8jOt6vLEHfe/JQGiRRSQQxSfFWpi1MquVdAyjUar5+76PVCmYl" crossorigin="anonymous"></script>
+
 <script src="{% static 'js/call.js' %}"></script>
 <script>
 
 
+let localStream;
+let peerConnection;
+let otherUser;
+let remoteRTCMessage;
+let localVideo = document.querySelector('#localVideo');
+
+let callInProgress = false;
+let iceCandidatesFromCaller = [];
+const constraints={
+    'video': true,
+    'audio': false, 
+}
 var callSocket = new WebSocket(
         'ws://'
         + window.location.host
         + '/ws/call/'
     );
 
-function beReady(){
-    console.log('I am ready')
+
+
+let pcConfig = {
+    "iceServers":
+        [
+            { "url": "stun:stun.jap.bloggernepal.com:5349" },
+            {
+                "url": "turn:turn.jap.bloggernepal.com:5349",
+                "username": "guest",
+                "credential": "somepassword"
+            },
```

Comment:
This can be removed, since we agreed to use only the googles open stun server, stated below in the code.